CHANGELOG.develop

```diff
@@ -715,6 +755,16 @@ is enabled
     - add ~SPC m s~ to save message as draft
     - add ~SPC m f~ to attach file
 - Replace mu4e multiple accounts with contexts (thanks to Allen Li)
+- Refactor to use =spacemacs/add-to-hooks= (thanks to Sylvain Benner)
+- Helm-mu4e integration and key bindings (thanks to Sébastien Bariteau and
```

Comment:
Helm-mu4e integration and key bindings - > Add Helm-mu4e integration and key bindings ?